bindings/c/test/mako/mako.cpp

```diff
@@ -181,38 +183,39 @@ int populate(Database db,
 	auto watch_trace = Stopwatch(watch_total.getStart());
 
 	Transaction systemTx = db.createTransaction();
-	for (int i = 0; i < args.tenants; ++i) {
-		while (true) {
-			// Until this issue https://github.com/apple/foundationdb/issues/7260 is resolved
-			// we have to commit each tenant creation transaction one-by-one
-			// while (i % 10 == 9 || i == args.tenants - 1) {
+	// This can be parameterized for mako or just left hardcoded to
+	// whatever value we find suitable
+	int batch_size = 10;
+	int batches = args.total_tenants / batch_size + 1;
```

Comment:
I think this still has an off-by-one when the number divides evenly. For example, for 10 tenants and a batch size of 10, this will give 2 batches. You could change this to:

```
int batches = math.ceil((double)args.total_tenants / batch_size);
```

or 

```
int batches = (args.total_tenants + batch_size - 1) / batch_size;
```